.github/workflows/cd.yml

```diff
@@ -0,0 +1,25 @@
+name: cd
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - uses: subosito/flutter-action@v1
+      - run: echo $CRENDTIALS > $FLUTTER_ROOT/.pub_cache/credentials.json
+      - run: flutter doctor -v
+      - run: flutter pub get
+      - run: flutter format --set-exit-if-changed --dry-run .
+      - run: flutter analyze --no-fatal-infos
+      - run: flutter test
+      - run: flutter pub publish --dry-run
+      - run: flutter pub publish
```

Comment:
```yaml
    steps:
      - uses: actions/checkout@v1
      - uses: sakebook/actions-flutter-pub-publisher@v1.3.1
        with:
          credential: ${{ secrets.CREDENTIAL_JSON }}
          dry_run: true
```

Easier and more readable

Reply:
If you want to use it, you can change it. The one you chose does not format and will cause pub points to drop: Use this instead https://github.com/marketplace/actions/dart-and-flutter-package-publisher

Reply:
Ok, we can also use https://github.com/marketplace/actions/dart-and-flutter-package-publisher

But what is our format check in CI for then?
It currently fails. So I'd say we'd rather stop CD from running if CI fails, fix the formatting issues within the repo and then run CD only if CI succeeds. Code in the repo should already be properly formatted, so there is no actual need to format it again before publish.

Reply:
We need to do `flutter format` in #33 to pass the ci. We just need to merge this to master then to #33 and fix whatever issues it raises.